solr/solr-ref-guide/modules/deployment-guide/pages/solr-control-script-reference.adoc

```diff
@@ -1191,11 +1199,10 @@ An absolute path may be supplied instead.
 +
 [%autowidth,frame=none]
 |===
-s|Required |Default: none
+|Optional |Default: `localhost:2181` if `ZK_HOST` env var not set
 |===
```

Comment:
These are not required per se, as if you have the env var or happen to run solr on default port, it is not required. So I switched to Optional in docs.